grammar/haxe.bnf

```diff
@@ -240,16 +243,20 @@ private ppToken ::= '#if' | "#elseif" | "#else" | "#end" | "#error" | "#line" |
  * after shiftRight so that it has priority.
  */
 shiftLeftOperator ::= '<<'
-shiftRightOperator ::= '>' '>' {pin=2}
-unsignedShiftRightOperator ::= '>' '>' '>' {pin=3}
+shiftRightOperator ::= <<shiftRight>>
+unsignedShiftRightOperator ::= <<unsignedShiftRight>>
+
+shiftLeftAssignOperator ::= '<<='
+shiftRightAssignOperator ::= <<shiftRightAssign>>
+unsignedShiftRightAssignOperator ::= <<unsignedShiftRightAssign>>
+
 shiftOperator ::= unsignedShiftRightOperator | shiftRightOperator | shiftLeftOperator
```

Comment:
I was looking at the PSI below.  I think we could make shiftOperator and shiftAssignOperator private.  They don't add much to the tree.  What do you think?  (This is NOT a request for a change, though you are free to make one if you agree. I will pull this without such a change.)